src/server/src/main/java/io/cassandrareaper/jmx/JmxProxyImpl.java

```diff
@@ -979,37 +979,28 @@ static Integer versionCompare(String str1, String str2) {
   }
 
   @Override
-  public void clearSnapshot(String repairId, String keyspaceName) throws ReaperException {
-    if (repairId == null || ("").equals(repairId)) {
+  public void clearSnapshot(String repairId, String keyspaceName) {
+    if (null == repairId || repairId.isEmpty()) {
       // Passing in null or empty string will clear all snapshots on the hos
       throw new IllegalArgumentException("repairId cannot be null or empty string");
     }
     try {
       ((StorageServiceMBean) ssProxy).clearSnapshot(repairId, keyspaceName);
-    } catch (IOException e) {
-      throw new ReaperException(e);
+    } catch (AssertionError | IOException e) {
+      LOG.error("failed to clear snapshot " + repairId + " in keyspace " + keyspaceName, e);
```

Comment:
Why are we swallowing the exception ? 
The UI will never know that the call failed and it will be required to dig in the logs to see what happened.

Reply:
See the description in the commit message:
> the AssertionError thrown by StorageService.clearSnapshot(..) when the snapshot can't be found is now logged as an error without throwing the ReaperException,

Clearing the snapshot only fails if the snapshot has since been deleted.
The UI does not need to provide an error message, the user requested the snapshot to be deleted and the snapshot has been deleted.
That's the rationale for not throwing the exception: it's an error that can be logged and kept internal, of no concern to the user.